boards/milkv/duo256m/doc/index.rst

```diff
@@ -0,0 +1,81 @@
+.. milkv_duo:
+
+Milk-V Duo 256M
+##########
```

Comment:
The underline is too short here and in the docs for the other 2 boards, see: https://github.com/zephyrproject-rtos/zephyr/actions/runs/8346902326/job/22845311762?pr=69594

You're also missing an index file, e.g. see this one: https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/sifive/index.rst